### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by christianonoh**
# I have completed the following tasks and would like to submit a pull request for your review:

- Set up linters: I have configured ESLint, Hint and Stylelint to ensure consistent code formatting and catch any potential syntax errors.

- Set up Webpack: I have implemented Webpack to bundle and optimize the project's assets, making it more efficient and easier to manage.

- Added markup with styling: I have added semantic HTML markup and CSS styling to improve the visual appeal and accessibility of the project.

- Used Gitflow: I have followed Gitflow branching model to ensure a structured workflow for the project and to make collaboration with other developers more efficient.

- Used ES6 modules: I have implemented ES6 modules to make the code more modular and reusable, improving maintainability and scalability.

Please review my changes and let me know if there are any further modifications needed.

## Thank you for your review.

### CODE REVIEW

File: src/modules/scoreboard.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      }
+    });
+    this.submitBtn.addEventListener('click', (event) => {
+      event.preventDefault();
+      this.addScore();
+    });
+  }
+
+  addScore() {
+    const userName = this.form.elements.name.value;
```
Comment (MaganaAsiati) [inline src/modules/scoreboard.js]: - You did a great job by implementing the functionality for adding items however it must be just a simple structure built in HTML. you will work on the implementation of functionality in the next project, At this stage, no API calls are required. :+1: 


:+1:

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @christianonoh 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/modules/scoreboard.js (hunk 1 above)
Comment (christianonoh) [reply src/modules/scoreboard.js]: Dear @MaganaAsiati ,

Thank you for your review. However I need some clarifications, am I supposed not to have any Javascript in the project ie just html and css?

**Review by Lameck1 — COMMENTED** [review]
Hello @christianonoh👋,

_This is an additional review due to the second opinion request._

I'm sorry to tell you that the code review is right. Please work within the scope of the milestone. That is, stick to the project requirements. All these attempts are actually a part of the project's requirement in an upcoming milestone. What do you expect the code reviewers to review in the next milestone when you do everything now? @christianonoh, the code review process doesn't work this way. Please do your work milestone by milestone and we won't be having these issues.

@MaganaAsiati, you were not supposed to review this PR in its current state. Instead, you were supposed to INVALIDATE this code review request. Please, keep this in mind.

@christianonoh, we are going to INVALIDATE @MaganaAsiati review so that you will be able to request a proper first review once you have ensured that your work is only within the scope of this milestone's project requirements.

Keep rocking 🚀!

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @christianonoh , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Good commit messages. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**